pkg/resources/aws.go

```diff
@@ -2047,3 +2048,46 @@ func FindELBName(tags []*elb.Tag) string {
 	}
 	return ""
 }
+
+// FilterUserProvidedItems removes user provided items such as subnets, and or the vpc.
+func FilterUserProvidedItems(resourceTrackers map[string]*ResourceTracker, clusterSpec api.ClusterSpec) {
```

Comment:
If someone can figure out a better way to iterate through this items, I am open.